packages/core/bootstrap/src/lib/ws/recorder.ts

```diff
@@ -0,0 +1,32 @@
+import WebSocket from 'ws'
+
+interface WsMessage {
+  type: 'sent' | 'received'
+  data: unknown
+}
+
+export class WsMessageRecorder {
+  private static messages: Array<WsMessage> = []
+
+  static add(message: WsMessage): void {
+    this.messages.push(message)
+  }
+
+  static print(): void {
+    console.log(`Recorded WebSocketMessages: ${JSON.stringify(this.messages)}`)
```

Comment:
Can you use the logger here?
```
import { logger } from '../external-adapter'
```

Reply:
The logger will print the inner json payload escaping strings, so I went with `console.log` to make it easier to copy and paste from the console to fixtures